src/control/zoom/ZoomControl.cpp

```diff
@@ -572,11 +579,17 @@ bool ZoomControl::onScrolledwindowMainScrollEvent(GtkWidget* widget, GdkEventScr
 	return false;
 }
 
-bool ZoomControl::onWidgetSizeChangedEvent(GtkWidget* widget, GdkRectangle *allocation, ZoomControl* zoom)
+bool ZoomControl::onWidgetSizeChangedEvent(GtkWidget* widget, GdkRectangle* allocation, ZoomControl* zoom)
 {
 	XOJ_CHECK_TYPE_OBJ(zoom, ZoomControl);
 
+	Rectangle r(allocation->x, allocation->y, allocation->width, allocation->height);
+	auto layout = gtk_xournal_get_layout(zoom->view->getWidget());
+	assert(widget != zoom->view->getWidget());
```

Comment:
Use `g_assert_true()` instead. We use it everywhere else so we shouldn't mix things.